PowerUp/app/src/main/res/layout-land/completed_scene.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Instead of nesting two LinearLayouts, it would be better to use a RelativeLayout